src/target/llvm/codegen_llvm.h

```diff
@@ -161,13 +161,6 @@ class CodeGenLLVM : public ExprFunctor<llvm::Value*(const PrimExpr&)>,
   void VisitStmt_(const EvaluateNode* op) override;
 
  protected:
-  /*! \brief The storage information */
-  struct StorageInfo {
-    /*! \brief The storage scope */
-    runtime::StorageScope scope;
-    /*! \brief The alignment of allocation */
```

Comment:
Let us keep the original data structure and leave alignment field here for now, in case we need some update in the future